Description: changed the name in lower case and deactivate installation
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-03-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/meson.build
+++ b/src/meson.build
@@ -663,7 +663,7 @@
   dependencies: dependencies,
   link_args: link_args,
   win_subsystem: 'windows',
-  install: true,
+  install: false,
   install_tag: 'common',
   c_args: ['-I.'],
 )
